examples/fuse_examples/multimodality/image_clinical/multimodality_image_clinical.ipynb

```diff
@@ -260,269 +202,32 @@
-    "                                   post_processing_func=post_cache_processing_func,\n",
-    "                                   augmentor=augmentor,\n",
-    "                                   visualizer=visualiser)\n",
-    "train_dataset.create()\n",
-    "```\n",
-    "\n",
-    "\n",
-    "Create pytorch dataloader:\n",
-    "```python\n",
-    "sampler = SamplerBalancedBatch(dataset=train_dataset, balanced_class_name='data.gt.gt_global.tensor', ...)\n",
-    "\n",
-    "train_dataloader = DataLoader(dataset=train_dataset,\n",
-    "                              batch_sampler=sampler, collate_fn=train_dataset.collate_fn, ...)\n",
-    "```\n"
+    "Details and instructions about our data package can be found [here](https://github.com/IBM/fuse-med-ml/blob/master/fuse/data/README.md)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "{'data': {'descriptor': 'ISIC_0072650', 'input': {'image': tensor([[[ 0.0946,  0.0697,  0.0780,  ...,  0.0972,  0.0895,  0.0539],\n",
-      "         [ 0.0949,  0.0330,  0.0297,  ...,  0.0238,  0.0538,  0.0739],\n",
-      "         [ 0.0570,  0.1089,  0.0800,  ...,  0.0573,  0.0514,  0.0621],\n",
-      "         ...,\n",
-      "         [ 0.0621,  0.0592,  0.0462,  ...,  0.0772,  0.0562,  0.0652],\n",
-      "         [ 0.0740,  0.0613,  0.0661,  ...,  0.0776,  0.0797,  0.0858],\n",
-      "         [ 0.0663,  0.0259,  0.0678,  ...,  0.0745,  0.0691,  0.0288]],\n",
-      "\n",
-      "        [[-0.0013,  0.0047,  0.0242,  ...,  0.0779,  0.0417,  0.1200],\n",
-      "         [ 0.0762,  0.1120,  0.1132,  ...,  0.0414,  0.0509,  0.0628],\n",
-      "         [ 0.0537,  0.0175,  0.0742,  ...,  0.0439,  0.0795,  0.0549],\n",
-      "         ...,\n",
-      "         [ 0.0746,  0.0749,  0.0897,  ...,  0.1027,  0.1242,  0.1001],\n",
-      "         [ 0.0691,  0.1050,  0.0385,  ...,  0.0946,  0.0794,  0.0999],\n",
-      "         [ 0.0417,  0.0608,  0.0409,  ...,  0.0711,  0.0661,  0.0838]],\n",
-      "\n",
-      "        [[ 0.0601,  0.1169,  0.0717,  ...,  0.1047,  0.1391,  0.0485],\n",
-      "         [ 0.1048,  0.0617,  0.0742,  ...,  0.0836,  0.0688,  0.0131],\n",
-      "         [ 0.0432,  0.1068,  0.0199,  ...,  0.1092,  0.0654,  0.0490],\n",
-      "         ...,\n",
-      "         [ 0.0260, -0.0077,  0.0777,  ...,  0.0723,  0.0479,  0.0839],\n",
-      "         [ 0.0470,  0.0423,  0.0591,  ...,  0.0384,  0.0935,  0.0933],\n",
-      "         [ 0.0553,  0.1287,  0.1373,  ...,  0.0868,  0.0698,  0.0035]]]), 'clinical': {'image': 'ISIC_0072650', 'age_approx': 30.0, 'anatom_site_general': 'anterior torso', 'lesion_id': 'BCN_0003356', 'sex': 'male'}}, 'gt': {'gt_global': {'tensor': tensor(1)}}}}\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "sample_index = 10\n",
     "print(train_dl.dataset[sample_index])"
```

Comment:
please correct me if I wrong, but I think this won't print the image and the clinical data together, if so, it could be nice to add it

Reply:
At this point, we don't have the image with the clinical data.